src/Umbraco.Cms.Integrations.SEO.SemrushTools/Umbraco.Cms.Integrations.SEO.SemrushTools/App_Plugins/UmbracoCms.Integrations/SEO/SemrushTools/semrush.resources.js

```diff
@@ -0,0 +1,50 @@
+﻿function semrushResources($http, umbRequestHelper) {
```

Comment:
For consistency with the CMS, the file name and function name should be singular - i.e. `semrush.resoure.js` and `semrushResource`.

Reply:
Item addressed.